Update level-two-cache.md (#42)
AdobeDocs/commerce-operations.en: help/configuration/cache/level-two-cache.md

>[!INFO]
>

> >For Adobe Commerce on cloud infrastructure, deploy variables can be used to configure L2 Cache Configuration [Deploy variables](https://devdocs.magento.com/cloud/env/variables-deploy.html#redis_backend) devdocs page.

Reviewer: ```suggestion
>For Adobe Commerce on cloud infrastructure, you can use [deploy variables](https://devdocs.magento.com/cloud/env/variables-deploy.html#redis_backend) for L2 cache configuration.
```
